Sources/Sugar/Middlewares/SimpleAuthMiddleware.swift

```diff
@@ -2,7 +2,7 @@ import HTTP
 import Vapor
 import AuthProvider
 
-internal final class SimpleAuthMiddleware: Middleware {
+final class SimpleAuthMiddleware: Middleware {
```

Comment:
This is the same - it should be marked `public`

Reply:
Sure!